service/controller/resource/monitoring/scrapeconfigs/test/aws/case-1-awsconfig.golden

```diff
@@ -49,7 +49,63 @@
   # Add role label.
   - source_labels: [__meta_kubernetes_node_label_role]
     target_label: role
+# Add scrape configuration for docker
```

Comment:
Do we want to add scrape config for Docker?

Reply:
This was based on a funky function that defaults .CAPICluster to true but I am not sure tbh considering we removed docker in a lot of places

Reply:
I'd say:
- either we make it "as it was before" (ie don't scrape docker)
- or we scrape docker because this was a bug, but it has to be shown in the changelog
...because it can have an impact on prometheus resources, and we want it to be easy to spot if anything goes wrong.

Reply:
Then let's not change docker for now :)

Reply:
I'm not sure about the versions so I'd rather not add any other change and we fix the issue